### ISSUE TITLE
"Install" state of (Steam) games often lost

### ISSUE THREAD

**Issue by lapo-luchini** [JosefNemec/Playnite#1362]
Since using 5.x I sometime have recently played games that instead of "Play" button have the "Install" button.
If I press it, after some half a second it becomes "Play" (without seemingly doing anything, but I guess Steam was called in the meantime?) and pressing it again properly launches the game.

**Comment by JosefNemec** [comment]
If the game doesn't properly report install state then it usually means that the game installation is corrupted in some way. Try verifying game files in the Steam client.

**Comment by lapo-luchini** [comment]
I'll try, but I doubt this can have happened to all my Steam games at the same time.
(some installed months ago and not used in the meantime)

**Comment by lapo-luchini** [comment]
1. verified game in Steam client
2. forced Steam library update in Playnite 5.6 (no warnings)
3. closed and re-opened Playnite
4. game still gray
5. tried "installing" another game
6. immediately became white
7. closed and re-opened Playnite
8. game installed in 5 still white, game Steam-checked in 1 still gray
It's nothing so bad for me, as I'm sorting by "last played" and not filtering out the uninstalled ones anyways, but if I can help debug further, please advise how.

**Comment by JosefNemec** [comment]
Sorry for late response. Can you go to `About Playnite` menu and create diag. package for me after reproducing the behavior you described in last comment? Thank you in advance

**Comment by lapo-luchini** [comment]
I just re-installed 5 games yesterday (that were installed already) and today they are no more.
(in the meanwhile the PC was hibernated, never shut off)
`4cdf51be-e02d-41c5-bae3-c9f8dde18548`

**Comment by JosefNemec** [comment]
Try reinstalling Steam client itself. This usually indicates that something is wrong with Steam installation and it must be repaired.

**Comment by lapo-luchini** [comment]
Sorry, I never re-installed Steam client from scratch (it's too much of an hassle and losing the installed state doesn't hit me so much, so I never found the will to do it) but I had various Steam client auto-updates over these months and this still happens to me even for games which were very recently installed. Feel free to close this bug if you don't need it to keep it.

**Comment by JosefNemec** [comment]
OK, I'm closing this. I'm 90% sure that this is somehow connected to broken Steam installation, since other people reported similar issues in past and clean Steam install resolved it.

**Comment by lapo-luchini** [comment]
It seems it was so, indeed.
As [suggested in this thread](https://steamcommunity.com/discussions/forum/1/530645446305521092/) I deleted everything in the Steam directory except `steam.exe` and the `steamapps/` directory, I re-executed Steam from Start menu, it automatically re-downloaded everything missing and started (with all my library intact).
I closed and re-opened Playnite (version 7.7) and, after a sync, all my installed Steam games are now properly detected!
